vxdev/setup-vxdev-machine.sh

```diff
@@ -0,0 +1,183 @@
+
+sudo ln -sf /var/vx/data /vx/data
+sudo ln -sf /var/vx/code /vx/code
+
+sudo ln -sf /vx/code/config/read-vx-machine-config.sh /vx/config/read-vx-machine-config.sh
+sudo ln -sf /home/vx/code/vxsuite-complete-system /vx/code/vxsuite-complete-system
+
+sudo chown -R vx:vx /vx/data
+sudo chmod -R ugo=rwX /vx/data
+
+sudo chown -R vx-admin:vx-admin /var/vx/admin
+sudo chmod -R u=rwX /var/vx/admin
+sudo chmod -R go-rwX /var/vx/admin
+
+sudo chown -R vx-admin:vx-admin /var/vx/config
+sudo chmod -R ugo=rwX /var/vx/config
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+sudo cp -rp \
+  "${DIR}/../run-scripts/run-kiosk-browser.sh" \
+  "${DIR}/../run-scripts/run-kiosk-browser-forever-and-log.sh" \
+  "${DIR}/../config" \
+  "${DIR}/../printing" \
+  "/vx/code"
+
+
+# Let vx read logs
+sudo usermod -aG adm vx
+
+# Set up log config
+sudo bash setup-scripts/setup-logging.sh
+
+
+# Let some users mount/unmount usb disks
+sudo usermod -aG plugdev vx
+
+# let vx manage printers
+sudo usermod -aG lpadmin vx
+
+sudo cp config/49-sane-missing-scanner.rules /etc/udev/rules.d/
+sudo usermod -aG scanner vx
+
+# make sure vx has pipenv
+sudo -u vx -i pip3 install pipenv
+
+# admin function scripts
+sudo ln -sf /vx/code/config/admin_bash_profile /vx/admin/.bash_profile
+sudo ln -sf /vx/code/config/admin-functions /vx/admin/admin-functions
+
+# machine manufacturer
+sudo sh -c 'echo "VotingWorks" > /vx/config/machine-manufacturer'
+
+# code version, e.g. "2021.03.29-d34db33fcd"
+sudo sh -c 'echo "$(date +%Y.%m.%d)-$(git rev-parse HEAD | cut -c -10)" > /vx/config/code-version'
+
+# code tag, e.g. "m11c-rc3"
+sudo sh -c 'git tag --points-at HEAD > /vx/config/code-tag'
+
+# machine ID
+sudo sh -c 'echo "0000" > /vx/config/machine-id'
+
+# sudo sh -c 'echo "MarkAndPrint" > /vx/config/app-mode'
+bash setup-scripts/setup-speech-synthesis.sh
+
+# vx-ui OpenBox configuration
+sudo mkdir -p /vx/ui/.config/openbox
+sudo ln -sf /vx/code/config/openbox-menu.xml /vx/ui/.config/openbox/menu.xml
+sudo ln -sf /vx/code/config/openbox-rc.xml /vx/ui/.config/openbox/rc.xml
+
+# If surface go, set proper resolution (1x not 2x)
+PRODUCT_NAME=`sudo dmidecode -s system-product-name`
+if [ "$PRODUCT_NAME" == "Surface Go" ]
+then
+    sudo ln -sf /vx/code/config/surface-go-monitors.xml /vx/ui/.config/monitors.xml
+fi
+
+# update sudoers file to give vx user special permissions
+sudo cp vxdev/sudoers /etc/sudoers
+
+# setup tpm2-totp
+sudo bash setup-scripts/setup-tpm2-totp.sh
+
+# setup signify
+sudo bash setup-scripts/setup-signify.sh
+
+# turn off time synchronization
+sudo timedatectl set-ntp no
+
+# Install app to configure VxDev
+sudo mkdir -p /vx/scripts
+sudo cp vxdev/update-vxdev.sh /vx/scripts/.
+sudo cp vxdev/update-vxdev.desktop /usr/share/applications/.
+gsettings set org.gnome.shell favorite-apps "['update-vxdev.desktop','firefox-esr.desktop', 'org.gnome.Nautilus.desktop', 'org.gnome.Terminal.desktop', 'gnome-control-center.desktop']"
```

Comment:
do we want Firefox as the default, or should we be grabbing chromium so it can be used to debug if kiosk-browser breaks?